panel/param.py

```diff
@@ -138,6 +139,11 @@ class Param(PaneBase):
     show_name = param.Boolean(default=True, doc="""
         Whether to show the parameterized object's name""")
 
+    sorted = param.Parameter(default=False, doc="""
```

Comment:
Could also be called `sort`.

I would like to use `param.ClassSelector` instead of `param.Parameter` to get validation. But I did not know how to set `class_=(bool, ?callable)`.

Reply:
I vote for `sort` and the parameter can be declared like this:

```python
from collections.abc import Callable
    
    sort = param.ClassSelector(class_=(bool, Callable))
```